pydantic/_internal/_generate_schema.py

```diff
@@ -87,6 +87,7 @@
     from ._schema_generation_shared import GetJsonSchemaFunction
 
 _SUPPORTS_TYPEDDICT = sys.version_info >= (3, 12)
+_AnnotatedType = type(Annotated[int, 123])
```

Comment:
Repeated calls to `type` and `Annotated.__class_getitem__` while building schemas seemed unnecessary/inefficient, but I didn't profile this or anything.